src/yarp.c

```diff
@@ -1,4 +1,5 @@
 #include "yarp.h"
+#include "sys/param.h"
```

Comment:
I think this is a POSIX header. I think for now I would just inline those functions.

Reply:
oof.

Reply:
Done.